CONTRIBUTING.md

```diff
@@ -215,7 +215,7 @@ Once satisfied with the example, [submit your pull request](https://help.github.
 
 ### Writing the exaple
 
-With a JavaScript example you start by creating a new `.html` file in `live-examples/js-examples`. The same naming convention applies here as it does for CSS. In this example we are going to contribute an example demonstrating the use of `Array.from` so, we'll create a new file called `array-from.html`.
+With a JavaScript example you start by creating a new `.html` file in subfolders of `live-examples/js-examples`. In this example we are going to contribute an example demonstrating the use of `Array.from` so, we'll create a new file called `array-from.html`. Since it is part of array, we're going to put it inside "array" folder.
```

Comment:
> With a JavaScript example you start by creating a new `.html` file in subfolders of `live-examples/js-examples`.

file in a subfolder of `live-examples/js-examples`.

> Since it is part of array, we're going to put it inside "array" folder.

Since it is part of the `Array` object, we're going to put it inside the "array" subfolder.